### PR TITLE
Upgrade/symfony4

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

**Comment by mickel1138** [comment]
Please give this a jolly good test, but the changes are minimal it seems.

**Comment by mickel1138** [comment]
PELAGOS-3760

**Comment by praneethpr** [comment]
To run it on local dev server 
bin/console server:run --env=dev --docroot=/path/to/web

File: src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php [hunk 1]
```diff
@@ -458,8 +458,9 @@ private function doesStringExist(array $metadataElement, string $comparisonEleme
      */
     private function sendEmail(): void
     {
-        $message = \Swift_Message::newInstance()
-            ->setSubject('DOI Sync Log - List of Dois that are out of sync')
+        $message = new \Swift_Message('DOI Sync Log - List of Dois that are out of sync');
+        $message
+            ->setSubject()
```
Comment (fightingtexasaggie) [inline src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php]: Wouldn't this wipe out the subject set via the Swift_Message constructor?

**Comment by praneethpr** [comment]
Filer is throwing an error on this branch
`filer.ERROR: Error processing dataset: Could not set NFS4 ACls to A::48:X for /home/ppondicherry/san/data/store/Y1.x098.000:0007 (Return value: 1) {"dataset_id":4398,"udi":"Y1.x098.000:0007","dataset_submission_id":13565} {"process_id":26892`

File: src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php (hunk 1 above)
Comment (mickel1138) [reply src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php]: This code has been fixed.

**Pull request closed without merging**